Block/Adminhtml/System/Config/Form/YotpoButton.php

```diff
@@ -51,7 +51,7 @@ protected function _getElementHtml(\Magento\Framework\Data\Form\Element\Abstract
      * @return string
      */
     public function getAjaxExportUrl()     { 
-        return '/admin/massmap/yotpocontroller/yotpocontroller/';
+        return $this->getUrl('massmap/yotpocontroller/yotpocontroller/', ['_secure' => $this->getRequest()->isSecure()]);
```

Comment:
don't we need the admin prefix?
why do we need the secure param?
which version of magento 2 is this? or are u working with?